src/_includes/layouts/base.njk

```diff
@@ -1,4 +1,4 @@
-{% set assetHash = global.random() %}
+{% set assetHash = global.hash() %}
```

Comment:
`Math.random()` can end up returning a value we've already used, so that could cause people to get a cached version of the file. Data-based seemed safer as it will always increment.